### ISSUE TITLE
[enhancement] [quality of life] double click on a keypair in Key ToolBox should opens key detail window of that keypair

### ISSUE THREAD

**Issue by fooryo** [saturneric/GpgFrontend#163]
double click on a keypair in Key ToolBox should opens key detail window of that keypair

**Comment by saturneric** [comment]
Currently, Key Details can be accessed from the Key ToolBox by clicking on the right mouse button to bring up a popup menu, is this still not very convenient?

**Comment by fooryo** [comment]
No, I really dont find it very convenient and that's because I often look at UIDs and subkeys statuses. 
In Kgpg I can expand every entry showing me all the IDs and subkey of that entry.
In Kleopatra there is the double click to open properties window.
Point to entry, right click, point to "Show Key Details" then click versus Point the entry and double click? Point and double click all the times. It really makes sense to me to have it. Even click on the entry to select and press enter could be cool btw.

But, hey, I'm talking about quality of life stuff. It's not that important to an extent.

Also, trying and learning gpgFrontend I dont understand what to do when a keyring has multiple valid subkeys for encryption and/or signing and I want to select that precise subkey to use. But I think that's for another issue

**Comment by saturneric** [comment]
Ok, I've also pondered whether GpgFrontend should also implement a tree-style Key ToolBox. This would have the advantage of expanding the details. But considering that it would interfere with the current basic operation logic, I'm going to wait and see for a while for now.

The second thing you mentioned, selecting specific subkeys to do some operations. This can be done with gpg, but unfortunately the library (gpgme) provided by gpg team does not provide this capability. I found some threads on the gpg developer mailing list about the problem, but strangely no one replied.
